modules/ding_place2book/ding_place2book.fields.inc

```diff
@@ -503,6 +507,22 @@ function ding_place2book_field_presave($entity_type, $entity, $field, $instance,
             ),
           );
         }
+
+        if (is_array($items) && isset($items[0]['event_id']) && isset($items[0]['event_maker_id'])) {
+          if ($items[0]['event_id'] == 1 && $items[0]['event_maker_id'] == 1) {
+            // This should not happen so we abort the saving off the node and alert the editor.
+
+            watchdog('ding_place2book', 'Original items: @original_items', array('@original_items' => print_r($settings , TRUE)), WATCHDOG_ERROR);
+            watchdog('ding_place2book', 'Synchronized items: @synchronized_items', array('@synchronized_items' => print_r($synchronized_items, TRUE)), WATCHDOG_ERROR);
+            watchdog('ding_place2book', 'Items to be saved: @items', array('@items' => print_r($items , TRUE)), WATCHDOG_ERROR);
```

Comment:
Is there a reason to log this error in multiple entries? If not then consider combining messages and data into a single watchdog entry.